src/Components/People.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+class People extends Component {
+  state = { people: [], input: "", inputPerson: {}, searched: false };
+
+  handleChange = (e) => {
+    this.setState({ input: e.target.value });
+  };
+
+  fetchAllPeople = async () => {
+    try {
+      const res = await axios.get(`https://ghibliapi.herokuapp.com/people`);
+      this.setState({ people: res.data });
+    } catch (error) {
+      this.setState({ people: [] });
+    }
+  };
+
+  componentDidMount = () => {
+    this.fetchAllPeople();
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { people, input } = this.state;
+    people.forEach((person) => {
```

Comment:
How will we know if we found the person or not? 

Try using find instead of forEach or adding different logic.